### PR TITLE
Make creating collection/partition follow tt mechanism

### PR DESCRIPTION
**Pull request by longjiquan**
Signed-off-by: longjiquan <jiquan.long@zilliz.com>
issue: #19175

### CODE REVIEW

**Comment by longjiquan** [comment]
rerun ut

**Comment by longjiquan** [comment]
rerun ut

File: internal/rootcoord/scheduler.go [hunk 1]
```diff
@@ -38,6 +47,7 @@ func newScheduler(ctx context.Context, idAllocator allocator.GIDAllocator, tsoAl
 		idAllocator:  idAllocator,
 		tsoAllocator: tsoAllocator,
 		taskChan:     make(chan task, n),
+		minDdlTs:     *atomic.NewUint64(0),
```
Comment (czs007) [inline internal/rootcoord/scheduler.go:50]: may be should initialized with MaxTimestamp?

File: internal/rootcoord/scheduler.go [hunk 2]
```diff
@@ -62,16 +73,33 @@ func (s *scheduler) execute(task task) {
...
 }
 
+func (s *scheduler) updateLatestTsoAsMinDdlTs() {
+	if len(s.taskChan) > 0 {
+		return
+	}
+
+	ts, err := s.tsoAllocator.GenerateTSO(1)
+	if err != nil {
+		log.Warn("failed to generate tso", zap.Error(err))
```
Comment (czs007) [inline internal/rootcoord/scheduler.go:97]: add more log info, indicate which function  call trigger this warning.

Comment (longjiquan) [reply internal/rootcoord/scheduler.go:97]: OK.

**Comment by longjiquan** [comment]
rerun ut

**Comment by longjiquan** [comment]
rerun ut

**Pull request merged**